src/components/TabBar/TabBar.jsx

```diff
@@ -7,23 +7,36 @@ const onActive = (tab) => {
   browserHistory.push(`${tab.route}${window.location.search}`);
 };
 
-const TabBar = ({ router, tabs, info }) => (
+const TabBar = ({ router, tabs, info, disabled = false }) => (
   <div className={styles.container}>
     <div className={styles.subContainer}>
       <Tabs
         inkBarStyle={{ backgroundColor: styles.blue }}
         className={styles.tabs}
         value={info}
       >
-        {tabs.map((tab, index) => (
-          <Tab
-            key={index}
-            className={styles.tab}
-            value={tab.name.toLowerCase()}
-            label={tab.name}
-            onActive={() => onActive(tab, router)}
-          />
-        ))}
+        {tabs.map((tab, index) => {
+          let isDisabled = false;
```

Comment:
I think instead of doing this here, it should be determined in the Match component when we generate the TabBar array.